src/xtgeo/well/well1.py

```diff
@@ -640,13 +649,13 @@ def to_roxar(self, *args, **kwargs):
 
 
         Args:
-            project (str, object): Magic string 'project' or file path to project
+            project (str): Magic string 'project' or file path to project
```

Comment:
Perhaps we should same "magic variable" since it is not a string in many cases (my bad, probably)